holoviews/core/dimension.py

```diff
@@ -158,6 +158,13 @@ def value_format(self, specs=None, **values):
     def range(self, specs=None, **values):
         return self._redim('range', specs, **values)
 
+    def label(self, specs=None, **values):
+        for k, v in values.items():
+            dim = self.parent.get_dimension(k)
```

Comment:
Seems like ``None`` is returned if there is no match for ``dim``? In which case a suitable exception/warning should be given if ``dim==None``.